playground/controllers/memo_controller.js

```diff
@@ -0,0 +1,33 @@
+import { Controller } from 'stimulus'
+import { useMemo } from 'stimulus-use'
+
+export default class extends Controller {
+  static memos = ['a', 'b']
+
+  connect() {
+    useMemo(this)
+
+    console.log(this.a)
+    console.log(this.b)
+    console.log(this.c)
+
+    console.log(this.a)
+    console.log(this.b)
+    console.log(this.c)
+  }
+
+  get a() {
+    console.log('getting a')
+    return 1
+  }
+
+  get b() {
+    console.log('getting b')
+    return 2
+  }
+
+  get c() {
+    console.log('getting c')
+    return 3
+  }
+}
```

Comment:
I think it would be nice to provide a more realistic example

suggestion 
```js
import { Controller } from 'stimulus'
import { useMemo } from 'stimulus-use'
import { intensiveNormalizeTask } from 'some-module'

export default class extends Controller {
  static memos = ['normalizedTex']
  static targets = ['longText']

  connect() {
    useMemo(this)

    this.normalizedText // process the text and cache the result
    this.normalizedText // gets the results from the cache
  }

  get normalizedText() {
    return intensiveNormalizeTask(this.longTextTarget.textContent)
  }
}
```